### DISCUSSION TITLE
Host Avalonia Control in WPF

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#13033] [Q&A] [answered]
I would like to try adding `ColorPicker` and ohter editor to the AvaloniaVS Xaml editor. How can you host Avalonia Control in WPF? 
I found an example of how to host avalonia Controls in WinForm that doesn't work.

**Comment by maxkatz6** [comment, marked as answer]
This control can be used https://github.com/maxkatz6/AvaloniaHwndHostSample/blob/main/Avalonia.Win32.Interoperability/AvaloniaHwndHost.cs
